api/src/main/java/com/codeforcommunity/dto/site/RecordStewardshipRequest.java

```diff
@@ -24,7 +24,7 @@ public RecordStewardshipRequest(
     this.weeded = weeded;
   }
 
-  private RecordStewardshipRequest() {}
+  protected RecordStewardshipRequest() {}
```

Comment:
nit: Maybe `default` would be better here instead?

Reply:
We should throw more keywords at it until it works like good Java programmers. /s